### DISCUSSION TITLE
Cloud computing

### DISCUSSION THREAD

**Discussion by Graham-Broughton** [plasma-umass/scalene#559] [Q&A] [answered]
Hey guys,
I haven't used scalene yet, but it looks awesome and I am excited to give it a whirl. So, my question is how would you go about using it in a cloud based VM? I don't know of a way to access the VM's localhost (assuming the html printout is localhost) from your local machine. I ask mainly because my system does not have a GPU so I wouldn't be able to accurately profile what the VM would be doing.
Thanks,
Graham

**Comment by emeryberger** [comment, marked as answer]
You can have Scalene emit a JSON file (`--json --outfile whatever.json`) and then upload it to the viewer here (all data remains local): http://plasma-umass.org/scalene-gui/.

**Reply by Graham-Broughton** [reply]
Awesome, thanks!
